pyiron/atomistics/structure/generator.py

```diff
@@ -112,6 +112,30 @@ def create_surface(
         return None
 
 
+def create_hkl_surface(lattice, hkl, layers, vacuum=1.0, center=False):
+    """
+    Create a surface with an arbitrary surface normal (hkl).
+
+    Args:
+        lattice: Atoms bulk instance or str, e.g. "Fe", from
```

Comment:
Small change in Docstrings: 

`lattice (pyiron.atomistics.structure.atoms.Atoms/str): Atoms bulk instance or str, e.g. "Fe", from`